src/components/SignUpSuccessForm/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const SignUpSuccessForm = () => {
+  return (
+    <section className="sign-up-success-form">
+      <header className="sign-up-success-form__header">Sample Picture</header>
+      <div className="sign-up-success-form__body">
+        <h2>Registration Success</h2>
+      </div>
```

Comment:
I think this suits better with `header` though since there's `h2` 🤔
```suggestion
      <div className="sign-up-success-form__header">
        <h2>Registration Success</h2>
      </div>
```